bootstrap/rpc/config.tf

```
@@ -21,6 +21,8 @@ resource "kubernetes_config_map_v1" "fabric_rpc_config" {
         auth0_client_secret = var.auth0_client_secret
         auth0_audience      = var.auth0_audience
         stripe_api_key      = var.stripe_api_key
+        email_ses_access_key_id = var.email_ses_access_key_id
+        email_ses_secret_access_key = var.email_ses_secret_access_key
```

Inside this `templatefile` function, you have to define the parameters needed to template the file. In this case, you are missing some:
```suggestion
        email_ses_access_key_id = var.email_ses_access_key_id
        email_ses_secret_access_key = var.email_ses_secret_access_key
        email_invite_ttl_min = var.email_invite_ttl_min
        email_ses_region = var.email_ses_region
        email_ses_verified_email = var.email_ses_verified_email
```

---

Do they need to be there even using default values in the main file?

---

ignore it, you are right!